src/Form/PasswordChangeType.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
You should create a new form for password changing instead of changing the form for resetting the password. This breaks password reset functionality